.github/workflows/ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           sudo apt-get clean
           df -h
       - uses: actions/checkout@v2
+      - uses: ./.github/actions/tune-runner-vm
```

Comment:
Please apply this only for phoenix job.

Reply:
What does a "the phoenix job" mean? I think it makes sense to apply the configuration to all jobs since there's no harm in doing that. 

The intention of "tune-runner-vm" action is to be able to add tuning for GitHub Actions Runner VMs. After the reverse name lookup problem has been fixed in GitHub, it might be able to leave the tune-runner-vm action in place, but just make it a no-op in case some other optimizations are added later. 

Some useful tuning to do in tune-runner-vm is to adjust other Linux kernel settings such as vm.swappiness or  THP settings, for example:
```
          echo 1 | sudo tee /proc/sys/vm/swappiness
          echo madvise | sudo tee /sys/kernel/mm/transparent_hugepage/enabled
```
Depending on the type of builds, these might be useful.